app/src/main/java/org/ole/planet/myplanet/model/RealmUserModel.java

```diff
@@ -68,8 +68,8 @@ public JsonObject serialize() {
         object.add("roles", getRoles());
         if (get_id().isEmpty()) {
             object.addProperty("password", getPassword());
-            object.addProperty("macAddress", NetworkUtils.getMacAddr());
-            object.addProperty("androidId", NetworkUtils.getMacAddr());
+            object.addProperty("macAddress", NetworkUtils.getUniqueIdentifier());
```

Comment:
Should this macAddress property be removed

Reply:
Yes removing all instances of mac address  since we are not using that onwards